### PR TITLE
new pbv lib

### PR DESCRIPTION
**Pull request by DDoucetHC**

### CODE REVIEW

File: projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html
Comment (linghanHC) [inline projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html:1]: where purposeOfCollection is used for the 3 pvb apps?  Remove it if not used in the 3 pvb apps.

File: tsconfig.json
Comment (linghanHC) [inline tsconfig.json:1]: The path is required if you use the pbv lib directly from the project folder/dist folder. It is not required if you use the packaged tgz pvb lib.  Remove it if the intention is to use the packaged versioned pvb lib.

File: projects/hpfb/pbv/package.json
Comment (linghanHC) [inline projects/hpfb/pbv/package.json:1]: 1. version it 1.0.0
2. add  "buildNumber": 0 (ref https://github.com/hres/AngularApps/blob/main-mono-repo/projects/hpfb/sdk/package.json). re-run the "pbv-lib:buildPackageAndMove" script then buildNumber will be incremented automatically

**Review by linghanHC — CHANGES_REQUESTED** [review]

File: projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html
Comment (DDoucetHC) [reply projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html:1]: i thought I wasnt supposed to delete anything from sdk?

Comment (linghanHC) [reply projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html:1]: My mistake. This comment is for "projects/hpfb/pbv/src/privacy-statement/privacy-statement.component.ts"

File: projects/hpfb/pbv/src/privacy-statement/privacy-statement.component.ts
Comment (linghanHC) [inline projects/hpfb/pbv/src/privacy-statement/privacy-statement.component.ts:1]: where purposeOfCollection is used for the 3 pvb apps? Remove it if not used in the 3 pvb apps.

Comment (DDoucetHC) [reply projects/hpfb/pbv/src/privacy-statement/privacy-statement.component.ts:1]: purpose of collection is used in the privacy notice statement for all the pbv forms and its a different text then the sdk ones if that's what you mean

Comment (linghanHC) [reply projects/hpfb/pbv/src/privacy-statement/privacy-statement.component.ts:1]: I'm saying do you use @Input() purposeOfCollection?; for pvb apps? if not, delete it

File: apps/pbv-rt/src/app/container/container.component.html
Comment (linghanHC) [inline apps/pbv-rt/src/app/container/container.component.html:1]: [purposeOfCollection]="'purpose.of.collection' | translate"  
is it needed for pvb rt app?
here you are passing in purpose.of.collection to the pvb lib privacy statement component, but what's the value of purpose.of.collection?

Comment (DDoucetHC) [reply apps/pbv-rt/src/app/container/container.component.html:1]: i believe it depends if we wanna keep the if else statement from before that sdk was using and the default template

Comment (linghanHC) [reply apps/pbv-rt/src/app/container/container.component.html:1]: if you are repeating the privacy statement from sdk, then why do we need a new one in pvb lib?
@AMckayHC what do you think?

Comment (DDoucetHC) [reply apps/pbv-rt/src/app/container/container.component.html:1]: its a different inside text which i guess we can use the same privacy statement and change the text in the json file

Comment (AMckayHC) [reply apps/pbv-rt/src/app/container/container.component.html:1]: Usually we do change the text in JSON if we can, maybe you and I can chat/review again?

File: projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html
Comment (linghanHC) [inline projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html:1]: don't think the if condition change is correct. have you tested?

Comment (DDoucetHC) [reply projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html:1]: i dont think ive tested with a json variable but it shows the template now which didnt before. ill test with json

Comment (linghanHC) [reply projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html:1]: test with MF too

Comment (DDoucetHC) [reply projects/hpfb/sdk/ui/information/privacy-statement/privacy-statement.component.html:1]: when i add a purpose.of.collection in the json it overrides the default and if nothing in json it used the default. I tested in mf-rt and pbv-rt

**Review by linghanHC — APPROVED** [review]

**Review by AMckayHC — APPROVED** [review]

**Pull request merged**